Cql/Cql.Compiler/LibrarySet.cs

```diff
@@ -210,16 +214,20 @@ private void RecalculateStateIfNecessary()
     /// Loads the specified library and its dependencies from the specified directory.
     /// </summary>
     /// <param name="elmDirectory">The directory containing the ELM files.</param>
-    /// <param name="lib">The name of the library to load.</param>
+    /// <param name="libraryPath">The name of the library to load.</param>
     /// <param name="version">The version of the library to load.</param>
     /// <returns>A collection of loaded libraries, including the specified library and its dependencies.</returns>
+    /// <remarks>Supply the name of the library in <paramref name="libraryPath"/>, not the name of the file.</remarks>
     public IReadOnlyCollection<Library> LoadLibraryAndDependencies(
         DirectoryInfo elmDirectory,
-        string lib,
+        string libraryPath,
```

Comment:
libraryName could be a better name for this parameter. It is not the whole path of it that is passed in here